metricbeat/module/elasticsearch/index/_meta/test/stats.700-alpha1.json

```diff
@@ -1,1473 +1,1601 @@
-                "suggest_time_in_millis" : 0,
-                "suggest_current" : 0
-            },
-            "merges" : {
-                "current" : 0,
-                "current_docs" : 0,
-                "current_size_in_bytes" : 0,
-                "total" : 0,
-                "total_time_in_millis" : 0,
-                "total_docs" : 0,
-                "total_size_in_bytes" : 0,
-                "total_stopped_time_in_millis" : 0,
-                "total_throttled_time_in_millis" : 0,
-                "total_auto_throttle_in_bytes" : 104857600
-            },
-            "refresh" : {
-                "total" : 26,
-                "total_time_in_millis" : 3076,
-                "listeners" : 0
-            },
-            "flush" : {
-                "total" : 0,
-                "periodic" : 0,
-                "total_time_in_millis" : 0
-            },
-            "warmer" : {
-                "current" : 0,
-                "total" : 17,
-                "total_time_in_millis" : 65
-            },
-            "query_cache" : {
-                "memory_size_in_bytes" : 0,
-                "total_count" : 0,
-                "hit_count" : 0,
-                "miss_count" : 0,
-                "cache_size" : 0,
-                "cache_count" : 0,
-                "evictions" : 0
-            },
-            "fielddata" : {
-                "memory_size_in_bytes" : 0,
-                "evictions" : 0
-            },
-            "completion" : {
-                "size_in_bytes" : 0
-            },
-            "segments" : {
-                "count" : 14,
-                "memory_in_bytes" : 60804,
-                "terms_memory_in_bytes" : 51071,
-                "stored_fields_memory_in_bytes" : 4368,
-                "term_vectors_memory_in_bytes" : 0,
-                "norms_memory_in_bytes" : 2304,
-                "points_memory_in_bytes" : 861,
-                "doc_values_memory_in_bytes" : 2200,
-                "index_writer_memory_in_bytes" : 4456672,
-                "version_map_memory_in_bytes" : 0,
-                "fixed_bit_set_memory_in_bytes" : 288,
-                "max_unsafe_auto_id_timestamp" : -1,
-                "file_sizes" : { }
-            },
-            "translog" : {
-                "operations" : 66,
-                "size_in_bytes" : 87028,
-                "uncommitted_operations" : 66,
-                "uncommitted_size_in_bytes" : 87028,
-                "earliest_last_modified_age" : 0
-            },
-            "request_cache" : {
-                "memory_size_in_bytes" : 0,
-                "evictions" : 0,
-                "hit_count" : 0,
-                "miss_count" : 2
-            },
-            "recovery" : {
-                "current_as_source" : 0,
-                "current_as_target" : 0,
-                "throttle_time_in_millis" : 0
-            }
+    "_all": {
+      "primaries": {
+        "docs": {
+          "count": 1257,
+          "deleted": 11
```

Comment:
We should probably "standardise" on the indentation of .json files. Do you use `.editorconfig`? https://github.com/elastic/beats/blob/master/.editorconfig#L15

Reply:
I'm guessing I don't right now. I'll look into it so formatting can be standardized.